### ISSUE TITLE
Seek to last few seconds of the an audio file stops the stream

### ISSUE THREAD

**Issue by sadhiappan** [muhku/FreeStreamer#133]
If you seek towards last few seconds on any audio file, the stream stops. This issue happens even on the demo project. Please let me know if you need more information. 

This project is fantastic and great contribution to the community. Keep up the good work!

**Comment by mmcdole** [comment]
@muhku I still seem to have this issue with certain files (although it does seem to be better).

[Here is a test mp3](https://dl.dropboxusercontent.com/u/638285/256346__jagadamba__robotic-droid-vocal-countdown-09.mp3)

If I attempt to seek to the last 15-20% of this 10 second MP3, the stream stops.

**Comment by muhku** [comment]
Yeah, #132 is related to this but it is not the same issue. I can push the podspec to the tree.

**Comment by muhku** [comment]
Pushed 87d4f6f35625c328da8ef212989599096747ae8e to the master. It basically extends the fix in #132 to correctly calculate the buffering override to the remaining seeked bytes. Let me know if it helps and I will tag and push the pod.
